<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labName">
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labUniversity">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>学校</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboUniversity">
       <item>
        <property name="text">
         <string>北京大学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>清华大学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>复旦大学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>浙江大学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>上海交通大学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>武汉大学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>华南理工大学</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labProvince">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>省份</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboProvince"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="labCity">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>城市</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboCity"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="labResult">
       <property name="text">
        <string>结果</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineResult">
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
